### ISSUE TITLE
Some unknown characters when chat with 7B

### ISSUE THREAD

**Issue by jinfagang** [antimatter15/alpaca.cpp#36]
<img width="968" alt="image" src="https://user-images.githubusercontent.com/21303438/226085265-09e2ab74-8c14-4646-b97d-03b92d797417.png">

Is this normal?

**Comment by artegoser** [comment]
it looks like the model is not properly trained. Dataset was not processed correctly. idk

![image](https://user-images.githubusercontent.com/59178854/226110248-da79cf68-cf65-4065-ab16-f3003c3320fc.png)

**Comment by tjohnman** [comment]
> it looks like the model is not properly trained. Dataset was not processed correctly. idk
> 
> ![image](https://user-images.githubusercontent.com/59178854/226110248-da79cf68-cf65-4065-ab16-f3003c3320fc.png)

This does not happen if you run it on llama.cpp. I don't know if it's part of the model's training as well, but you can find those "Instruction"/"Response" hardcoded in the alpaca.cpp source.

**Comment by xuyuliang** [comment]
Yeah , It's normal.  Because your console is  coded GBK by default  ( you are using Chinese version Windows ). Change it to utf-8 would be ok ( I just did it ) 
In PowerShell , type "chcp 65001" firest before you type chat.exe to start alpaca.
